README.md

```diff
@@ -150,6 +150,8 @@ render((
 ), document.getElementById("app"));
 ```
 
+WARNING: The `formData` field is **not** used only for initialization but it sets the data of the form if your wrapping component rerenders. Therefore make sure you listen to the `onChange` event and update the data you pass to the `formData` field.
```

Comment:
Sounds good. Should it be added before the example along with the text on the top or leave it here under the example?